enterprise_access/apps/api_client/license_manager_client.py

```diff
@@ -0,0 +1,26 @@
+"""
+API client for calls to the license-manager service.
+"""
+from django.conf import settings
+
+from enterprise_access.apps.api_client.base_oauth import BaseOAuthClient
+
+
+class LicenseManagerApiClient(BaseOAuthClient):
+    """
+    API client for calls to the license-manager service.
+    """
+    api_base_url = settings.LICENSE_MANAGER_URL + '/api/v1/'
+    subscriptions_endpoint = api_base_url + 'subscriptions/'
```

Comment:
Should we set these as config options instead of inline?

Reply:
probably not. LICENSE_MANAGER_URL makes sense  to be in settings, but the expected paths from that service should always be the same. if we ever update LM or ecommerce to use api/v2, then we can make a change here then

Reply:
(in particular cause these paths are used nowhere but here. and if it's any consolation, that's how it is in LM)

Reply:
👍